src/main/java/io/jenkins/plugins/localization_zh_cn/CommunityDecorator.java

```diff
@@ -37,14 +37,8 @@ public boolean showCommunityLink() {
             return isCurrentLanguage;
         }
 
-        UserCommunityProperty.ShowConditions condition = UserCommunityProperty.ShowConditions.Chinese;
-        try {
-            String conditionName = communityProperty.getShowCondition();
-            condition = UserCommunityProperty.ShowConditions.valueOf(conditionName);
-        } catch (IllegalArgumentException | NullPointerException e) {
-            // ignore the invalid conditions
-            // when this plugin just be installed without restart, conditionName could be null
-        }
+        String conditionName = communityProperty.getShowCondition();
+        UserCommunityProperty.ShowConditions condition = UserCommunityProperty.ShowConditions.valueOf(conditionName);
```

Comment:
Could you provide the context for these changes? I cannot figure it out by reading the code.

Reply:
It was spotbugs